### PR TITLE
Add WildFly 14.0.0.Final

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

File: imagestreams/wildfly-centos7.json [hunk 1]
```diff
@@ -167,6 +167,26 @@
...
+          "version": "14.0"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/openshift/wildfly-140-centos7:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        },
+        "name": "14.0"
```
Comment (bparees) [inline imagestreams/wildfly-centos7.json]: please update the "latest" imagestream tag to point to this new tag.  (line 24)

Comment (jamesnetherton) [reply imagestreams/wildfly-centos7.json]: Whoops! I forgot about that.

Updated in 745debd11cb233cc0ed73ac0bc21409a18953858.

**Pull request merged**